force-app/main/default/classes/AccountTriggerHelper.cls

```diff
@@ -0,0 +1,28 @@
+public class AccountTriggerHelper {
+
+public static void beforeInsert(List<Account> newAccounts) {
+    for (Account acc : newAccounts) {
+        if(acc.Type == null){
+            acc.Type = 'Prospect';
+            } 
+            acc.BillingStreet = acc.ShippingStreet;
+            acc.BillingCity = acc.ShippingCity;
+            acc.BillingState = acc.ShippingState;
+        
+            acc.BillingPostalCode = acc.ShippingPostalCode;
+            acc.BillingCountry = acc.ShippingCountry;
+
+            if (acc.Phone != NULL && acc.Website!= NULL && acc.Website != NULL) {
+                acc.Rating = 'Hot';
+                    }
+    }
+}
```

Comment:
Good work here, but mind your formatting and indentation on the curly braces {}. They should align with the indentation of where they start.